lib/ice_cube/builders/string_builder.rb

```diff
@@ -13,14 +13,18 @@ def piece(type, prefix = nil, suffix = nil)
     end
 
     def to_s
-      @types.each_with_object(@base || '') do |(type, segments), str|
+      string = @base || ''
+      @types.each do |type, segments|
```

Comment:
Would love to but I don't have push rights on your fork :P I'll add them to gocardless/ice_cube (along with French translation) and PR them into your fork.